### ISSUE TITLE
Phpdoc: cannot list references

### ISSUE THREAD

**Issue by przepompownia** [phpactor/phpactor#1521]
To reproduce: 
- from `lib/Phpactor.php` got to `Resolver` imported there
- add some virtual property, e.g.
```php
/**
 * @property int $foo
 */
class Resolver
```
- in `Phpactor` after constructing `Resolver` add `$masterSchema->foo` and go to it (currently completion and going works for me)
- from the virtual property definition to display references to it

**Comment by dantleech** [comment]
do you mean to invoke the LSP references on the virtual property? that's not something i've considered but possible.

**Comment by przepompownia** [comment]
Yes, exactly. For example, Yii has classes (named active records) that reflect table in SQL databases with access to its columns through magic properties. I would like then to list accesses to a specific column.
